code/modules/projectiles/guns/projectile/boltgun.dm

```diff
@@ -103,6 +103,7 @@
 	matter = list(MATERIAL_STEEL = 20, MATERIAL_WOOD = 10)
 	wielded_item_state = "_doble_wood"
 	spawn_blacklisted = FALSE
+	gun_parts = list(/obj/item/part/gun = 3 ,/obj/item/stack/material/steel = 16)
```

Comment:
Novakovic doesn't even uses plasteel yet dismantling the gun parts will give us some, let's stick to thermodynamics laws and won't create matter out of nothing. It's so irredemeably shitty that it shouldn't yield any. Same for kardashev-mosin.